src/site/stages/build/process-cms-exports/transformers/node-press_release.js

```diff
@@ -23,15 +24,20 @@ const transform = entity => ({
     ? mapKeys(entity.fieldAddress[0], (v, k) => camelCase(k))
     : null,
   fieldIntroText: getDrupalValue(entity.fieldIntroText),
-  fieldOffice:
-    entity.fieldOffice && entity.fieldOffice[0]
-      ? { entity: entity.fieldOffice[0] }
-      : null,
-  fieldPdfVersion: entity.fieldPdfVersion[0] || null,
+  fieldOffice: entityObjectForKey(entity, 'fieldOffice'),
+  fieldPdfVersion: entityObjectForKey(entity, 'fieldPdfVersion'),
   fieldPressReleaseContact: entity.fieldPressReleaseContact.map(i => ({
     entity: i,
   })),
-  fieldPressReleaseDownloads: entity.fieldPressReleaseDownloads,
+  fieldPressReleaseDownloads: entity.fieldPressReleaseDownloads.map(i => ({
+    entity: {
+      ...i,
+      image: {
+        ...i.image,
+        url: encodeURI(i.image.url.replace('public:/', '/img')),
+      },
+    },
+  })),
```

Comment:
You don't need this change

Reply:
Still need to insert the `entity` key into each object in the `fieldPressReleaseDownloads` array, but yeah, no need for the `replace` on the url any more.